client/state/data-getters/index.js

```diff
@@ -230,3 +230,33 @@ export const requestSiteAlerts = siteId => {
 		}
 	);
 };
+
+export const requestSiteScan = siteId => {
+	const requestId = `site-scan-${ siteId }`;
+
+	return requestHttpData(
+		requestId,
+		http(
+			{
+				method: 'GET',
+				path: `/sites/${ siteId }/scan`,
+				apiNamespace: 'wpcom/v2',
+			},
+			{}
+		),
+		{
+			freshness: 5 * 60 * 1000,
```

Comment:
Is this the same as 5min?

Reply:
We no longer need this data-getter because we are using the `QueryJetpackScan` component to fetch the data and load it into the store.